tox.ini

```diff
@@ -1,7 +1,8 @@
 [tox]
 minversion = 2.4
+envlist = py{2.7,3.6}
```

Comment:
I'd rather not specify the Python versions here, as some environments may not have these versions. Let's just specify the target versions in Travis and let tox run against the local python version.